package.json

```diff
@@ -1,9 +1,26 @@
 {
   "dependencies": {
-    "husky": "^9.1.6"
+    "@typescript-eslint/eslint-plugin": "^8.10.0",
+    "@typescript-eslint/parser": "^8.10.0",
+    "eslint": "^8.57.1",
```

Comment:
I think we could move all these files to the frontent directory.  This way we don't have to repeat a lot of things

Reply:
We can do that, but then only the frontend files will be checked.
And the way husky works is it checks for the staged files during pre-commit.
As git is initialized at the root directory, we have to run linters from there which will require these packages to be present at the root.

The one thing that can be done is move the repeated files from the frontend to the base, as they get checked anyhow from the pre-commit

Reply:
1. Only check the frontend project is enough.
2. We should keep the root directory content to be clean
3. We can first let the pre commit check work on the frontend and actually run it

Reply:
Got your point, but the way husky works is from a git initialized location.
1. If aiming for frontend, then git has to be initialized there, then we can run husky from there